instrumentation/aws-sdk/aws-sdk-1.11/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v1_11/SqsMessage.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.awssdk.v1_11;
+
+import java.util.Map;
+
+interface SqsMessage {
+
+  Map<String, String> getAttributes();
```

Comment:
Is the purpose of this interface to avoid having muzzle blow up on the actual request type? If so, can you add a comment about that?

Reply:
done